mmdeploy/backend/tensorrt/utils.py

```diff
@@ -205,18 +206,18 @@ def from_onnx(onnx_model: Union[str, onnx.ModelProto],
         max_shape = param['max_shape']
         profile.set_shape(input_name, min_shape, opt_shape, max_shape)
     if config.add_optimization_profile(profile) < 0:
-        logger.warning(f'Invalid optimization profile {profile}.')
+        trtlogger(f'Invalid optimization profile {profile}.')
```

Comment:
This might be a warning instead of a log according to the level of `log_level`?

Reply:
We can not control the log level.
If we set  log_level=trt.Logger.ERROR,
when we set logger.log(trt.Logger.Warning, 'message'), 
it take no effects.

Reply:
The code line `logger.warning(f'Invalid optimization profile {profile}.')` is just to give a warning to the user. So, give a warning `logger.log(trt.Logger.Warning, 'message')`.

Reply:
I'm not sure here since if the user use the log level *INFO*, the warning here would be info then. Or, we can use mm logger here to do warning? @RunningLeon what's your opinion?

Reply:
> I'm not sure here since if the user uses the log level _INFO_, the warning here would be info then. Or, we can use mm logger here to do warning? @RunningLeon what's your opinion?

Agree, better use logger from mmdeploy.